expose-daemon.ps1

```diff
@@ -0,0 +1,149 @@
+$ErrorActionPreference = 'Stop';
+
+if (!(Test-Path $env:USERPROFILE\.docker)) {
+  mkdir $env:USERPROFILE\.docker
+}
+
+$ipAddresses = ((Get-NetIPAddress -AddressFamily IPv4).IPAddress) -Join ','
+
+function ensureDirs($dirs) {
+  foreach ($dir in $dirs) {
+    if (!(Test-Path $dir)) {
+      mkdir $dir
+    }
+  }
+}
+
+# https://docs.docker.com/engine/security/https/
+# Thanks to @artisticcheese! https://artisticcheese.wordpress.com/2017/06/10/using-pure-powershell-to-generate-tls-certificates-for-docker-daemon-running-on-windows/
+function createCA($serverCertsPath) {
```

Comment:
Not a blocker and I like the dynamic approach but maintainability cost are increased by doing this in Windows and I think some static certs could do the same job.

Reply:
I'll file a follow-up issue for this

Reply:
#226